action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: "'workflow' or 'repository' (dispatch_type is deprecated and will be removed)"
     required: false
     default: "workflow"
+  fail_on_findings:
+    description: "fail control when finding is found"
+    required: false
+    default: true
```

Comment:
not sure what this actually changes but this PR is changing the default value (entrypointwise).

Tried to see when this envvar is set (didn't find). The code in the entry point regarding it is:
`fail_on_findings = str(environ.get('FAIL_ON_FINDINGS', 'false')).lower() == 'true'`
which means that the default value currently is `False` but here we change it to `True` are we sure things still working as expected?

Reply:
@avichayJit It should not have any consequence. Basically the logics is the following:
* If no env_var is passed, then default is False
* If env_var is passed, use the value of the env_var (default value for the env_var is True)